MIDIchlorians/MIDIchlorians/AudioManager.swift

```diff
@@ -29,14 +29,20 @@ struct AudioManager {
 
     //call this to play audio with single directory
     //returns success
-    func play(audioDir: String) -> Bool {
+    func play(audioDir: String, bpm: Int? = nil ) -> Bool {
         guard let audioID = audioDict[audioDir] else {
             return false
         }
         AudioClipPlayer.playAudioClip(soundID: audioID)
         return true
     }
 
+    func hackCheckValidIndex(row: Int, col: Int) -> Bool {
```

Comment:
called it hack, cus it's used for the hard coding, and it shouldn't be in this class LOL